ppa/archive/admin.py

```diff
@@ -114,6 +116,32 @@ def bulk_add_collection(self, request, queryset):
                                              'to collections')
     bulk_add_collection.allowed_permissions = ('change',)
 
+    def suppress_works(self, request, queryset):
```

Comment:
i like this naming (`works` to me implies multiple/a queryset) but wonder if it should be consistent with `bulk_add_collection`. should we just rename that one to `add_works_to_collection`? does this even matter?

Reply:
Agree the names are not consistent; I like your suggestion, but also not sure how much it matters...

I'll change if it's easy.